DpmCore/ruckus.tcl

```diff
@@ -6,7 +6,7 @@ set family [getFpgaFamily]
 
 # Check for valid FPGA 
 if { $::env(PRJ_PART) != "XC7Z045FFG900-2" && $::env(PRJ_PART) != "XCZU15EG-FFVC900-1-E" } {
-   puts "\n\nERROR: PRJ_PART must be either XC7Z045FFG900-2 or XCZU15EG-FFVC900-1-E in the Makefile\n\n"; exit -1
+#   puts "\n\nERROR: PRJ_PART must be either XC7Z045FFG900-2 or XCZU15EG-FFVC900-1-E in the Makefile\n\n"; exit -1
```

Comment:
I think you accidentally left this.  Why did you comment it out?

Reply:
I can't run a full cob simulation with it there. Xilinx requires one and only one project part, but the full sim project consists of both DPMs and a DTM. My HPS full sim also has a FEB FPGA.

Reply:
So when I do `make vcs` for the project that is the full sim, it has a project part that is not one of the DTM parts. I need it to work with this.